collection/collection.go

```diff
@@ -369,6 +373,18 @@ func (s *localSet) List(ctx context.Context, pid profile.ID, lp params.List) ([]
 
 	results := make([]dsref.VersionInfo, 0, lp.Limit)
 
+	if len(lp.OrderBy) != 0 {
+		switch lp.OrderBy[0] {
+		case "updated":
+			sortedCol := make([]dsref.VersionInfo, len(col))
+			copy(sortedCol, col)
+			sort.Sort(byUpdated(sortedCol))
+			col = sortedCol
+		case "name":
+		default:
+		}
+	}
```

Comment:
We could have a map of order keys/sort func's to make this slimmer but not worth it for just a handful of keys.